$NetBSD: patch-Cargo.toml,v 1.6 2023/07/28 20:47:55 pin Exp $

Don't pull from a specific pull-request.
Patch tree_magic_mini crate within pkgsrc.

--- Cargo.toml.orig	2023-07-26 22:12:55.000000000 +0000
+++ Cargo.toml
@@ -52,7 +52,7 @@ owo-colors = "3.5.0"
 wu-diff = "0.1.2"
 # rayon 1.7 requires rust 1.59
 rayon = "1.6.1"
-tree_magic_mini = "3.0.2"
+tree_magic_mini = "3.0.3"
 # bumpalo 3.12 requires rust 1.60
 bumpalo = "3.11.1"
 unicode-width = "0.1.9"
@@ -81,9 +81,3 @@ lto = "thin"
 [[bin]]
 name = "difft"
 path = "src/main.rs"
-
-[features]
-
-[patch.crates-io]
-# https://github.com/mbrubeck/tree_magic/pull/13 hasn't been merged yet.
-tree_magic_mini = { git = 'https://github.com/Wilfred/tree_magic', branch = 'fix-panic-for-empty-strings' }
